internal/impl/global.go

```diff
@@ -123,31 +121,25 @@ func (d *Devbox) addFromPull(pullCfg *Config) error {
 	return d.AddGlobal(diff...)
 }
 
-func GlobalConfigPath() (string, error) {
-	home, err := os.UserHomeDir()
-	if err != nil {
-		return "", errors.WithStack(err)
-	}
-	return filepath.Join(home, "/.config/devbox/"), nil
+func GlobalDataPath() string {
+	path := xdg.DataSubpath(filepath.Join("devbox/global", currentGlobalProfile))
+	_ = os.MkdirAll(path, 0755)
+	return path
 }
 
-func globalProfilePath() (string, error) {
-	configPath, err := GlobalConfigPath()
-	if err != nil {
-		return "", err
-	}
-	nixDirPath := filepath.Join(configPath, "nix")
-	_ = os.MkdirAll(nixDirPath, 0755)
-	return filepath.Join(nixDirPath, "profile"), nil
+func GlobalNixProfilePath() string {
+	return filepath.Join(GlobalDataPath(), "nix-profile")
 }
 
 // Checks if the global profile is in the path
 func ensureGlobalProfileInPath() error {
-	profilePath, err := globalProfilePath()
-	if err != nil {
-		return err
+	currentPath := xdg.DataSubpath("devbox/global/current")
+	// For now default is always current. In the future we will support multiple
+	// and allow user to switch.
+	if err := os.Symlink(GlobalDataPath(), currentPath); err != nil && !os.IsExist(err) {
+		return errors.WithStack(err)
 	}
-	binPath := filepath.Join(profilePath, "bin")
+	binPath := filepath.Join(currentPath, "nix-profile", "bin")
```

Comment:
Could we have the symlink point to the nix profile (`GlobalNixProfilePath`) to keep the path shorter when they add the path in their rc file?

(They can add `.../devbox/global/current/bin`)

Or are they not adding that to the path and instead doing something like `source $(devbox global activate)`?